src/evict/evict_lru.c

```diff
@@ -97,6 +97,11 @@ __evict_entry_priority(WT_SESSION_IMPL *session, WT_REF *ref)
 
     read_gen += btree->evict_priority;
 
+/*
+ * For pages that are getting random updates (often index pages), try not to reconcile them too
+ * often. It makes better use of I/O if they accumulate more changes between reconciliations
+ */
+#define WT_EVICT_MODIFY_COUNT_MIN 15 /* Number of modifications since the prior reconciliation */
```

Comment:
This define belongs down with the code that uses it, now that you have removed this skew proportion of the change.